client/my-sites/post-share/index.jsx

```diff
@@ -104,20 +158,104 @@ class PostShare extends Component {
 		);
 	}
 
-	dismiss = () => {
-		this.props.dismissShareConfirmation( this.props.siteId, this.props.post.ID );
-	};
+	renderShareButton() {
+		const { siteId, translate } = this.props;
+		return (
+			<div className="post-share__button-actions">
+				<Button onClick={ this.previewSharingPost }>
+					{ translate( 'Preview' ) }
+				</Button>
+
+				<ButtonGroup className="post-share__share-combo">
+					<Button
+						className="post-share__button"
+						primary
+						onClick={ this.sharePost }
+						disabled={ this.isButtonDisabled() }
+					>
+						{ translate( 'Share post' ) }
+					</Button>
+
+					<CalendarButton
+						primary
+						className="post-share__schedule-button"
+						title={ translate( 'Set date and time' ) }
+						tabIndex={ 3 }
+						siteId={ siteId }
+						popoverPosition="bottom left" />
+				</ButtonGroup>
+			</div>
+		);
+	}
 
-	sharePost = () => {
-		this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message );
-	};
+	renderUpgradeNudge() {
+		if ( this.isSchedulingEnabled ) {
+			return null;
+		}
 
-	isButtonDisabled() {
-		if ( this.props.requesting ) {
-			return true;
+		const {
+			businessDiscountedRawPrice,
+			businessRawPrice,
+			translate,
+			userCurrency,
+		} = this.props;
+
+		return (
+			<Banner
+				className="post-share__footer-banner"
+				callToAction={
+					translate( 'Upgrade for %s', {
+						args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'To upload themes, upgrade to Business Plan' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).